py-polars/polars/cfg.py

```diff
@@ -64,6 +64,14 @@ def set_tbl_cols(cls, n: int) -> "Type[Config]":
         ----------
         n
             number of columns to print
+        
+        Examples
+        --------
+
+        >>> import polars as pl
```

Comment:
In the docstring examples, we assume `import polars as pl` has already been done, so this line can be removed.